packages/cli/src/templates/snow/hooks/write-trellis-context.py

```
@@ -0,0 +1,639 @@
+                try:
+                    obj = json.loads(line)
+                    if isinstance(obj, dict):
+                        for key in ("summary", "title", "id", "path", "message", "status"):
+                            val = obj.get(key)
+                            if isinstance(val, str) and val.strip():
+                                summary = val.strip()
+                                break
+                        else:
+                            summary = json.dumps(obj, ensure_ascii=False)
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)